bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Yep! Good call, but leave exactly one trailing slash if the prefix is non-empty, and drop one of the slashes from the URL. That way when the string is empty we get only one `/`, and when it's present, things work out correctly.